### PR TITLE
support ICagent in cce cluster

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

support ICagent in cce cluster

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud' TESTARGS='-run TestAccCCEClusterV3_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud -v -run TestAccCCEClusterV3_basic -timeout 360m -parallel 4
=== RUN   TestAccCCEClusterV3_basic
=== PAUSE TestAccCCEClusterV3_basic
=== CONT  TestAccCCEClusterV3_basic
--- PASS: TestAccCCEClusterV3_basic (482.37s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       482.424s
```

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_cce_cluster_v3.go [hunk 1]
```diff
@@ -382,6 +383,15 @@ func resourceCCEClusterV3Create(d *schema.ResourceData, meta interface{}) error
...
 	d.SetId(create.Metadata.Id)
 
+	icAgentClient, err := config.AomV1Client(GetRegion(d, config))
+
+	installParam := icagents.InstallParam{
+		ClusterId: d.Id(),
+		NameSpace: "default",
+	}
+
+	_ = icagents.Create(icAgentClient, installParam)
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_cce_cluster_v3.go]: suggest to check the result and log the error message.

File: huaweicloud/endpoints.go [hunk 2]
```diff
@@ -87,6 +87,10 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Version:          "api/v3",
 		WithOutProjectID: true,
 	},
+	"aom": {
+		Name:    "aom",
+		Version: "svcstg/icmgr/v1",
+	},
```
Comment (ShiChangkuo) [inline huaweicloud/endpoints.go:93]: we should add the catalog in acc test.

File: huaweicloud/resource_huaweicloud_cce_cluster_v3.go (hunk 1 above)
Comment (Jason-Zhang9309) [reply huaweicloud/resource_huaweicloud_cce_cluster_v3.go]: fixed

File: huaweicloud/endpoints.go (hunk 2 above)
Comment (Jason-Zhang9309) [reply huaweicloud/endpoints.go:93]: fixed

**Pull request merged**